Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in airtable.gemspec
 gemspec
+
+gem 'pry'
```

Comment:
Is pry needed here?

Reply:
Its for development purposes. Really helpful. Gemfile is not pushed to the gem file so used only for development environment of gem file only.